lib/authorization.ts

```diff
@@ -285,9 +232,9 @@ const getActorMask = async (
 	scope: JsonSchema = {},
 ): Promise<JsonSchema> => {
 	const permissionFilters = await getRoleViews(context, backend, actor);
-	let mask = await applyMarkers(context, backend, actor, {
-		type: 'object',
 
+	let actorPermissionMask = await applyMarkers(context, backend, actor, {
```

Comment:
Try to keep changes unrelated to the PR outside the PR. Some fixes here and there is fine but as far as I can see all changes to this file are unrelated